frontend/src/services/http/HttpResponseInterceptors.ts

```diff
@@ -13,10 +14,14 @@ const MAX_RETRY = 2
 // --- Interceptors
 
 // Assume it is caught in error block based on default handling.
-async function forbiddenStatusItcpt(error: any) {
+async function accessErrorResponsesItcpt(error: any) {
+    if (error.response.status == 401 && (retryCount.value < MAX_RETRY)) {
+        return refreshTokenAndReTry(error.request.responseURL, error.response);
+    }
     // 403 special handling; until MAX_RETRY reached.
```

Comment:
Since we change to 401, please change comment to 401.
Thanks Nick.

Reply:
My bad... Thanks for spotting it.